rasa/server.py

```diff
@@ -377,6 +377,128 @@ def validate_request_body(request: Request, error_message: Text) -> None:
         raise ErrorResponse(HTTPStatus.BAD_REQUEST, "BadRequest", error_message)
 
 
+def events_request_format_spec() -> Dict[Text, Any]:
+    """Expected request body schema used for validation."""
+    return {
```

Comment:
that's the same schema as for the actions module, isn't it? How about deduplicating that?

Reply:
Yes, good idea! I can extract the events subschema - however where would I place it, as a global variable in `action.py`?

Reply:
How about creating a new module in `rasa.shared.utils.schemas`?

Reply:
Done 😃